projects/ngx-sub-form/src/lib/ngx-root-form.component.spec.ts

```diff
@@ -10,22 +10,22 @@ import { DataInput } from './ngx-sub-form.decorators';
 interface Vehicle {
   color?: string | null;
   canFire: boolean | null;
-  numberOfPeopleOnBoard: number | null;
+  numberOfCrewMembersOnBoard: number | null;
 }
 
-const MIN_NUMBER_OF_PEOPLE_ON_BOARD = 5;
-const MAX_NUMBER_OF_PEOPLE_ON_BOARD = 15;
+const MIN_NUMBER_OF_CREW_MEMBERS_ON_BOARD = 5;
+const MAX_NUMBER_OF_CREW_MEMBERS_ON_BOARD = 15;
 
 const getDefaultValues = (): Required<Vehicle> => ({
   color: '#ffffff',
   canFire: true,
-  numberOfPeopleOnBoard: 10,
+  numberOfCrewMembersOnBoard: 10,
```

Comment:
Should I rename that to `nbCrewMembers`?

Reply:
I did rename a bunch of these, I suspect I missed this one - I named it `crewMemberCount` everywhere else so I think we should do the same.